source/cdk/cdk_slurm_stack.py

```diff
@@ -1122,7 +1135,7 @@ def create_file_system(self):
 
             subnet_ids = [self.subnet.subnet_id]
             if self.config['slurm']['storage']['ontap']['deployment_type'] == 'MULTI_AZ_1':
-                for subnet in self.vpc.private_subnets:
+                for subnet in self.subnets:
```

Comment:
I don't like this name. Rename to the more descriptive self.private_and_isolated_subnets